administrator/language/de-DE/plg_multifactorauth_email.ini

```
@@ -0,0 +1,28 @@
+; Joomla! German Translation
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MULTIFACTORAUTH_EMAIL="Multi-Faktor-Authentifizierung - Authentifizierungscode per E-Mail"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_FORCE_ENABLE_DESC="Sollte der Authentifizierungscode per E-Mail automatisch als Option für <em>alle</em> Benutzer hinzufügt werden? Nützlich, um Benutzern, die den Zugang zu ihrem Hauptauthentifikator verloren haben und keine Kopie der Backup-Codes aufbewahrt haben, eine Ausweichmöglichkeit zu bieten, allerdings auf Kosten eines gewissen Maßes an Kontrolle und Sicherheit."
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_FORCE_ENABLE_LABEL="Aktivieren erzwingen"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_120="Zwei Minuten (empfohlen)"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_180="Drei Minuten"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_300="Fünf Minuten"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_30="Eine halbe Minute"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_60="Eine Minute"
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_DESC="Der Code wird im gewählten Zeitraum neu erstellt. Bitte beachten, dass ein generierter Code mindestens so lange und höchstens doppelt so lange gültig ist. Je länger diese Zeitspanne ist, desto wahrscheinlicher ist es, dass der Code mit roher Gewalt erzwungen werden kann, und desto unsicherer ist die Website. Ein Zeitraum von 2 Minuten ist ein guter Kompromiss zwischen Benutzerfreundlichkeit und Sicherheit."
+PLG_MULTIFACTORAUTH_EMAIL_CONFIG_TIMESTEP_LABEL="Zeitraum der Code-Erzeugung"
+PLG_MULTIFACTORAUTH_EMAIL_EMAIL_BODY="Multi-Faktor-Authentifizierung auf {SITENAME}. Der Authentifizierungcode lautet {CODE}."
+PLG_MULTIFACTORAUTH_EMAIL_EMAIL_SUBJECT="Der {SITENAME} - Authentifizierungscode lautet -{CODE}-"
+PLG_MULTIFACTORAUTH_EMAIL_ERR_INVALID_CODE="Ungültiger oder abgelaufener Code. Bitte die Seite neuladen, um einen neuen Code zu senden. Sicherstellen, dass der Code innerhalb von zwei Minuten nach der Anforderung des Codes eingeben wird."
+PLG_MULTIFACTORAUTH_EMAIL_LBL_DISPLAYEDAS="Authentifizierungscode per E-Mail"
+PLG_MULTIFACTORAUTH_EMAIL_LBL_LABEL="Authentifizierungscode"
+PLG_MULTIFACTORAUTH_EMAIL_LBL_PRE_MESSAGE="Es wurde ein sechsstelliger Code für die Multi-Faktor-Authentifizierung per E-Mail gesendet. Bitte diesen unten eingeben."
+PLG_MULTIFACTORAUTH_EMAIL_LBL_SETUP_PLACEHOLDER="Sechsstelliger Authentifizierungscode"
+PLG_MULTIFACTORAUTH_EMAIL_LBL_SHORTINFO="Erhalt von sechsstelligen Authentifizierungscodes per E-Mail."
```

```suggestion
PLG_MULTIFACTORAUTH_EMAIL_LBL_SHORTINFO="Sechsstelligen Code per E-Mail erhalten"
```

Der Punkt am Ende der Zeile entfällt, da kein vollständiger Satz